datafusion/core/src/lib.rs

```diff
@@ -382,13 +382,13 @@
 //!
 //! Calling [`execute`] produces 1 or more partitions of data,
 //! as a [`SendableRecordBatchStream`], which implements a pull based execution
-//! API. Calling `.next().await` will incrementally compute and return the next
+//! API. Calling [`.next()`]`.await` will incrementally compute and return the next
 //! [`RecordBatch`]. Balanced parallelism is achieved using [Volcano style]
 //! "Exchange" operations implemented by [`RepartitionExec`].
 //!
 //! While some recent research such as [Morsel-Driven Parallelism] describes challenges
 //! with the pull style Volcano execution model on NUMA architectures, in practice DataFusion achieves
-//! similar scalability as systems that use morsel driven approach such as DuckDB.
+//! similar scalability as systems that use pull driven schedulers such as DuckDB.
```

Comment:
I think DuckDB is push-based. https://github.com/duckdb/duckdb/issues/1583

Reply:
Yes, you are right -- sorry I will fix this

Reply:
Fixed in 368bfed32826e530c920b429aa3541884e19e132